### PR TITLE
Assignment1 - Conner Hanson

### PR DESCRIPTION
**Pull request by connerhanson**
Fixed application crashing and made all unit tests pass.

### CODE REVIEW

**Comment by JohanneJayde** [comment]
I can review this one

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -21,9 +21,15 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        double percent = ((double)numberCorrectAnswers / numberOfQuestions) * 100;
+    	return $"{percent}%";
```
Comment (JohanneJayde) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:25]: Interesting choice for fixing the issue with the percentage. I never thought about using string interpolation to convert the double to a string. I like it more than appending a percentage to convert it to a string.

Comment (JohanneJayde) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:25]: Interesting choice for fixing the issue with the percentage. I never thought about using string interpolation to convert the double to a string. I like it more than appending a percentage to convert it to a string.

**Review by JohanneJayde — COMMENTED** [review]
Assignment Details
Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit  (solo, doesn't apply)
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌
Issue 3: Feature request unit tested ❌

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -79,13 +85,15 @@ public static Question[] LoadQuestions(string filePath)
 
             string correctAnswerIndex = lines[lineIndex + 4];
 
-            Question question = new();
+            Question question = new Question();  //Added Question(); to intialize
```
Comment (BenjaminMichaelis) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:88]:
```suggestion
            Question question = new()
```
While this is not incorrect, the general rule of thumb is to use the simplified new();. But that can (as with practically every coding guideline) be argued.

https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0090?WT.mc_id=8B97120A00B57354

File: README.md [hunk 3]
```diff
@@ -11,16 +11,24 @@ Next Week the reading is Chapters 4-6 so schedule accordingly. (It is a lot of r
...
+  * Issue 3: Add a new feature request implemented ❌
+  * Issue 3: Feature request unit tested ❌
 
-See [Docs](https://github.com/IntelliTect-Samples/EWU-CSCD371-2024-Winter/blob/main/Docs)
\ No newline at end of file
+See [Docs](https://github.com/IntelliTect-Samples/EWU-CSCD371-2024-Winter/blob/main/Docs)
+
+## Execution 
+
+* Done on Linux command line, without the use of an IDE.
```
Comment (BenjaminMichaelis) [inline README.md:30]: Impressive! :)

**Pull request closed without merging**